filters.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: future_fstrings -*-
+
+#    Friendly Telegram (telegram userbot)
+#    Copyright (C) 2018-2019 The Authors
+
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from .. import loader, utils
+
+import logging
+logger = logging.getLogger("FilterModule")
+
+
+def register(cb):
+    cb(Filters())
+
+
+class Filters(loader.Module):
+    """When you filter a text, it auto responds to it if a user triggers the word)"""
+
+    def __init__(self):
+        self.name = _("Filters")
+        self._me = None
+        self._ratelimit = []
+
+    async def client_ready(self, client, db):
+        self._db = db
+        self._client = client
+        self._me = await client.get_me()
+
+    async def filtercmd(self, message):
+        """Adds a filter into the list."""
+        args = (utils.get_args_raw(message)).strip()
+        reply = await message.get_reply_message()
+        chatid = str(message.chat_id)
+        filters = self._db.get("FilterModule", "filters")
```

Comment:
Use __name__ ffs

Reply:
what.. i didnt know i did a pr on filters too. name should stay that way btw

Reply:
The name must change.